bin/nib-ls

```diff
@@ -180,6 +184,22 @@ def proc_file(f, opts):
     else:
         row += [ '' ]
 
+    if opts.header_fields:
+        # signals "all fields"
+        if opts.header_fields == '*':
+            # TODO: might vary across file types, thus prior sensing would be needed
+            header_fields = h.keys()
+        else:
+            header_fields = opts.header_fields.split(',')
```

Comment:
raising error -- I would prefer to avoid it... at times I just run nib-ls on a bunch of files of different types, so if in some type some header field is missing -- oh well.  Similar logic here is for other types (e.g. no qform/sform in [sq]form in AnalyzeHeader, but otherwise I print 'error'. Ok -- can print 'error' here as well for consistency I guess